includes/class-wc-stripe-account.php

```
@@ -419,14 +419,63 @@ public function is_webhook_enabled() {
+	 * @since 10.5.0
+	 * @return array List of Stripe webhook event types.
+	 */
+	public static function get_webhook_events() {
+		$events = self::WEBHOOK_EVENTS;
+
+		if ( WC_Stripe_Feature_Flags::is_agentic_commerce_enabled() ) {
+			$events[] = 'checkout.session.completed';
+		}
+
+		return $events;
+	}
```

PHPMD (2.15.0)

[error] The method get_webhook_events is not named in camelCase. (undefined)

(CamelCaseMethodName)